networktest.go

```diff
@@ -187,6 +192,13 @@ func (l *lightningNode) Start(lndError chan error) error {
 	var errb bytes.Buffer
 	l.cmd.Stderr = &errb
 
+	// If the -showoutput flag is passed to the test, we show node output
+	if *showOutput {
+		w := io.MultiWriter(&errb, os.Stderr)
```

Comment:
Have you tested this with race condition turned on? Last time I did this I had to re-invent thread-safe writer.